src/views/Room.vue

```diff
@@ -29,7 +29,10 @@
         <v-layout row justify-space-between>
           <v-flex column md8 sm12>
             <v-tabs grow v-model="queueTab">
-              <v-tab>Queue</v-tab>
+              <v-tab>
+                Queue
+                <span class="bubble">{{$store.state.room.queue.length < 99? $store.state.room.queue.length : "99+"}}</span>
```

Comment:
Good job on finding where the room state is. Just a couple of nitpicks:

1. If there are are 99 videos in the room, then it shows "99+". In this situation, I think it should say "99" instead. Change `< 99` to `<= 99` 
2. The spacing is a little bit inconsistent with the rest of the code. There should be a space after the `{{` and before the `}}`, and both sides of the `?` should have a space.